drivers/classic/src/get_global_param.c

```diff
@@ -46,26 +46,12 @@ get_global_param(FILE *gp)
     char                       optstr[MAXSTRING];
     char                       flgstr[MAXSTRING];
     char                       flgstr2[MAXSTRING];
-    int                        file_num;
+    size_t                     file_num;
     int                        field;
     int                        i;
-    int                        tmpstartdate;
-    int                        tmpenddate;
-    int                        lastvalidday;
-    int                        lastday[] = {
-        31,     /* JANUARY */
-        28,     /* FEBRUARY */
-        31,     /* MARCH */
-        30,     /* APRIL */
-        31,     /* MAY */
-        30,     /* JUNE */
-        31,     /* JULY */
-        31,     /* AUGUST */
-        30,     /* SEPTEMBER */
-        31,     /* OCTOBER */
-        30,     /* NOVEMBER */
-        31,     /* DECEMBER */
-    };
+    unsigned                   tmpstartdate;
+    unsigned                   tmpenddate;
```

Comment:
Be explicit about the size of `unsigned`. I know it defaults to `unsigned int`, but since the rest of VIC is explicit about that, stick with it.